projects/plugins/jetpack/modules/search/class-jetpack-instant-search.php

```diff
@@ -730,24 +740,25 @@ public function save_old_sidebars_widgets( $old_sidebars_widgets = null ) {
 	 */
 	public function remove_wp_migrated_widgets( $sidebars_widgets ) {
 		// Hook the action only when it is a theme switch i.e. $this->old_sidebars_widgets is not empty.
-		if ( empty( $this->old_sidebars_widgets ) ) {
+		// Some extra checks to make sure the hook has minmized affection.
+		if (
```

Comment:
Could we add additional tests for this conditional in `test-class-jetpack-instant-search.php`?

Reply:
added in de51e15